content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,188 @@
 
-Get an overview of how many requests each team is closing per day in order to measure their
-performance. Identify productivity levels to understand how many requests they are able to handle.
+Click on an individual rating to see additional details about the rating assigned by the customer,
+including a link to the original ticket.
 
-.. image:: reports/report_close_date_team.png
+.. image:: reports/reports-ratings-details.png
    :align: center
-   :alt: View of a helpdesk ticket analysis by team and close date in Odoo Helpdesk
+   :alt: View of the details of an individual customer rating
 
 .. tip::
-   Filter it by *Assignee* to see Key Performance Indicators (KPI) per agent.
+   Select :guilabel:`Visible Internally Only` to hide a rating from the customer portal.
+
+The :guilabel:`Customer Ratings` report is displayed in a kanban view by default, but can also be
+displayed in graph or pivot view.
+
+.. seealso::
+   - :doc:`ratings`
+
+Choosing the view and filters
+=============================
+
+The best view to display a report will depend on the what data is being measured, and how it needs
+to be grouped. See below for additional information on the available views for the
+:guilabel:`Helpdesk` reports.
+
+.. note::
+   Only one measure may be selected at a time for graphs, while pivot tables can include multiple
+   measures.
+
+Using the pivot view
+--------------------
 
-- The number of hours tickets are taking to be solved, grouped by team and ticket type.
+The *pivot* view presents data in an interactive manner. All three :guilabel:`Helpdesk` reports are
+available in the pivot view.
 
-Check if your expectations are met by *measuring* the *Time to close (hours)*. Your
-customers not only expect fast responses but they also want their issues to be handled quickly.
+Switch a report to the pivot view by selecting the icon at the top right of the screen.
 
-.. image:: reports/report_time_to_close.png
+.. image:: reports/reports-pivot-view.png
    :align: center
-   :alt: View of helpdesk ticket analysis of the hours to close by ticket type and team in
-         Odoo Helpdesk
+   :alt: View of the SLA status analysis report in Odoo Helpdesk
 
-Save filters
-============
+To add a group to a row or column, click the plus button (:guilabel:`+`) next to :guilabel:`Total`,
+and then select one of the groups. To remove one, click the minus button (:guilabel:`-`) and
+de-select.
 
-Save the filters you use the most and avoid having to reconstruct them every time they are
-needed. To do so, set the groups, filters, and measures needed. Then, go to *Favorites*.
+Using the graph view
+--------------------
 
-.. image:: reports/favorites.png
+The *graph* view presents data in either a *bar*, *line*, or *pie* chart. Switch to the graph view
+by selecting the icon at the top right of the screen. To switch between the different chart views,
+select the appropriate icon at the top left of the chart.
+
+.. tabs::
+
+   .. tab:: Bar chart
```

Comment:
```suggestion
   .. tab:: Bar graph
```

Reply:
I know it seems counterintuitive (since it is the "graph view") but when you hover over the individual icons for each of these (bar, line, pie) they are referred to as "charts" and not graphs. I realize it might not seem correct, however since it follows the UI, I'd rather match that. (I included the labels in the screenshots for that reason as well).

If it seems out of place I don't mind changing it, but I wanted to match the UI